### ISSUE TITLE
Call to a member function getLocalization() on null

### ISSUE THREAD

**Issue by dantleech** [sulu-io/sulu#957]
When a node has no localizations, then the localization ParentChildFinder breaks:

````
{"error":{"code":500,"message":"Internal Server Error","exception":[{"message":"Error: Call to a member function getLocalization() on null","class":"Symfony\\Component\\Debug\\Exception\\FatalErrorException","trace":[{"namespace":"","short_class":"","class":"","type":"","function":"","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/vendor\/sulu\/sulu\/src\/Sulu\/Component\/Content\/Mapper\/LocalizationFinder\/ParentChildAnyFinder.php","line":104,"args":[]},{"namespace":"Sulu\\Component\\Content\\Mapper\\LocalizationFinder","short_class":"ParentChildAnyFinder","class":"Sulu\\Component\\Content\\Mapper\\LocalizationFinder\\ParentChildAnyFinder","type":"->","function":"getAvailableLocalization","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/vendor\/sulu\/sulu\/src\/Sulu\/Component\/Content\/Mapper\/LocalizationFinder\/ChainFinder.php","line":42,"args":[]},{"namespace":"Sulu\\Component\\Content\\Mapper\\LocalizationFinder","short_class":"ChainFinder","class":"Sulu\\Component\\Content\\Mapper\\LocalizationFinder\\ChainFinder","type":"->","function":"getAvailableLocalization","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/vendor\/sulu\/sulu\/src\/Sulu\/Component\/Content\/Mapper\/ContentMapper.php","line":1307,"args":[]},{"namespace":"Sulu\\Component\\Content\\Mapper","short_class":"ContentMapper","class":"Sulu\\Component\\Content\\Mapper\\ContentMapper","type":"->","function":"loadByNode","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/vendor\/sulu\/sulu\/src\/Sulu\/Component\/Content\/Mapper\/ContentMapper.php","line":959,"args":[]},{"namespace":"Sulu\\Component\\Content\\Mapper","short_class":"ContentMapper","class":"Sulu\\Component\\Content\\Mapper\\ContentMapper","type":"->","function":"loadByParentNode","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/vendor\/sulu\/sulu\/src\/Sulu\/Component\/Content\/Mapper\/ContentMapper.php","line":930,"args":[]},{"namespace":"Sulu\\Component\\Content\\Mapper","short_class":"ContentMapper","class":"Sulu\\Component\\Content\\Mapper\\ContentMapper","type":"->","function":"loadByParent","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/ContentBundle\/Repository\/NodeRepository.php","line":248,"args":[]},{"namespace":"Sulu\\Bundle\\ContentBundle\\Repository","short_class":"NodeRepository","class":"Sulu\\Bundle\\ContentBundle\\Repository\\NodeRepository","type":"->","function":"getNodes","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/vendor\/sulu\/sulu\/src\/Sulu\/Bundle\/ContentBundle\/Controller\/NodeController.php","line":272,"args":[]},{"namespace":"Sulu\\Bundle\\ContentBundle\\Controller","short_class":"NodeController","class":"Sulu\\Bundle\\ContentBundle\\Controller\\NodeController","type":"->","function":"cgetAction","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/bootstrap.php.cache","line":3022,"args":[]},{"namespace":"","short_class":"","class":"","type":"","function":"call_user_func_array:{\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/bootstrap.php.cache:3022}","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/bootstrap.php.cache","line":3022,"args":[]},{"namespace":"Symfony\\Component\\HttpKernel","short_class":"HttpKernel","class":"Symfony\\Component\\HttpKernel\\HttpKernel","type":"->","function":"handleRaw","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/bootstrap.php.cache","line":2984,"args":[]},{"namespace":"Symfony\\Component\\HttpKernel","short_class":"HttpKernel","class":"Symfony\\Component\\HttpKernel\\HttpKernel","type":"->","function":"handle","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/bootstrap.php.cache","line":3133,"args":[]},{"namespace":"Symfony\\Component\\HttpKernel\\DependencyInjection","short_class":"ContainerAwareHttpKernel","class":"Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel","type":"->","function":"handle","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/bootstrap.php.cache","line":2377,"args":[]},{"namespace":"Symfony\\Component\\HttpKernel","short_class":"Kernel","class":"Symfony\\Component\\HttpKernel\\Kernel","type":"->","function":"handle","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/web\/admin.php","line":44,"args":[]},{"namespace":"","short_class":"","class":"","type":"","function":null,"file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/config\/router_admin.php","line":23,"args":[]},{"namespace":"","short_class":"","class":"","type":"","function":"{main}","file":"\/home\/daniel\/www\/sulu-io\/sulu-standard\/app\/config\/router_admin.php","line":0,"args":[]}]}]}}
````

**Comment by danrot** [comment]
There has already been some discussion at https://github.com/sulu-io/sulu-standard/issues/220

**Comment by chirimoya** [comment]
@dantleech ... still an issue?

**Comment by danrot** [comment]
@dantleech How can a node have no localizations at all?

**Comment by wachterjohannes** [comment]
i think we have this issue with `i18n:-title` - no locale in property name. maybe this causes something like this?

**Comment by danrot** [comment]
You are right, I think that happens for all `StructureExtension`s, right?

**Comment by wachterjohannes** [comment]
hmm the last time i have also title but that could be because of old data! currently i only found extension properties!

**Comment by danrot** [comment]
Does that really have to solved in 1.1? Looks like an edge case to me, and as soon as we have fixed https://github.com/sulu-io/sulu/issues/1704, this shouldn't be the case anymore. I think there is more important stuff for the release.

**Comment by webmozart** [comment]
@chirimoya What should we do about this? Close?
